### ISSUE TITLE
Error downloading a ckpt

### ISSUE THREAD

**Issue by UriPad** [TheLastBen/fast-stable-diffusion#450]
When i try to download a ckpt it show me the next message 
![Captura de Pantalla 2022-11-10 a la(s) 4 58 01](https://user-images.githubusercontent.com/117908700/201073594-e489a9a7-af9d-43ca-9753-3f246f87c046.png)

**Comment by TheLastBen** [comment]
WD CKPT ?

**Comment by UriPad** [comment]
Sorry? english is not mine first language... :c
what i mean is that, when i try to charge a model.ckpt for retraining that pops up

**Comment by UriPad** [comment]
> WD CKPT ?

sorry, english isn't my first language.
what i mean is that, when i try to charge a model.ckpt for retraining that pops up

**Comment by UriPad** [comment]
> > WD CKPT ?
> 
> sorry, english isn't my first language. what i mean is that, when i try to load a model.ckpt for retraining that pops up

even with another colab it show me this
![Captura de Pantalla 2022-11-10 a la(s) 16 47 43](https://user-images.githubusercontent.com/117908700/201222307-38489cea-f4fe-427c-ad9e-eaf2cdf1089a.png)

**Comment by cansakirt** [comment]
If you already have the model checkpoint in your Google Drive, skip that cell completely and run these:
```
# Install missing module
!pip install pytorch_lightning
# Select your model checkpoint
CKPT_Path = "/content/gdrive/MyDrive/Modelos/Raa.ckpt"
# Convert the model checkpoint
!python /content/diffusers/scripts/convert_original_stable_diffusion_to_diffusers.py --checkpoint_path "{CKPT_Path}" --dump_path /content/stable-diffusion-v1-5
```
This will create a folder `/content/stable-diffusion-v1-5` with the converted checkpoint.
After this is done, you can run the cell titled **_Create/Load a Session_**

---
### Optional:
If you plan on reusing the same model at a later date, you can backup this converted checkpoint folder to Google Drive:
`!cp -R /content/stable-diffusion-v1-5/* /content/gdrive/MyDrive/AI/stable_diffusion/stable-diffusion-v1-5/`

Next time you're running Dreambooth, instead of converting the model again, you can copy the already converted model from Google Drive to Colab filesystem:
```
!pip install pytorch_lightning
!mkdir /content/stable-diffusion-v1-5/
!cp -R /content/gdrive/MyDrive/AI/stable_diffusion/stable-diffusion-v1-5/* /content/stable-diffusion-v1-5/
```

**Comment by TheLastBen** [comment]
I don't think you need to do any of the steps above, just make sure you're using the latest colab ; https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Comment by UriPad** [comment]
> 
Yeah, i try that when i was locking for solutions, but sadly no... even with your other recommendation didn't work, but thanks

**Comment by UriPad** [comment]
> I don't think you need to do any of the steps above, just make sure you're using the latest colab ; https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

i thought that the colab in the main page it was the latest colab, this work perfectly, thank you

**Comment by TheLastBen** [comment]
that link is the one from the main page, avoid saving the colab to you gdrive, or at least, use the link from time to time to keep it updated
